lib/browser/guest-window-manager.js

```diff
@@ -224,7 +224,10 @@ ipcMainInternal.on('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_OPEN', (event, url, fra
   if (features == null) features = '';
 
   const disposition = 'new-window';
-  const { options, additionalFeatures } = convertFeaturesString(features, frameName);
+  const { options, webPreferences, additionalFeatures } = parseFeatures(features);
+  if (!options.title) options.title = frameName;
+  options.webPreferences = webPreferences;
```

Comment:
same comment as above; slightly more obtuse now but will make more sense in a later refactoring PR.